envs/dev/templates/team-carmen.yaml

```diff
@@ -1,21 +0,0 @@
-apiVersion: argoproj.io/v1alpha1
```

Comment:
why are we dropping these?

Reply:
@kcoleman731  - The file is not part of the teams referenced on the readme of this repo nor the main SSP workshop repo.  I want to make sure that the workshop teams matches the teams also on this repo...

Reply:
Team carmen is in the readme? https://github.com/aws-samples/ssp-eks-workloads/blob/main/README.md?plain=1#L50

Team carmen also provides and example of an Application pointing to a remote source url (i.e. not in this repo) so it is a good example to keep. 

I think it is ok if the workshop only references two of the four teams that are in this repo.

Reply:
Ok added the `carmen` manifests under `dev` environments, and to the readme on this repo.